service/src/main/java/uk/gov/hmcts/reform/fpl/service/email/content/AbstractEmailContentProvider.java

```diff
@@ -40,6 +51,19 @@ protected AbstractEmailContentProvider(String uiBaseUrl) {
             .put("caseUrl", uiBaseUrl + "/case/" + JURISDICTION + "/" + CASE_TYPE + "/" + caseDetails.getId());
     }
 
+    protected ImmutableMap.Builder<String, Object> getSDOPersonalisationBuilder(CaseDetails caseDetails, CaseData caseData) {
+        return ImmutableMap.<String, Object>builder()
+            .put("familyManCaseNumber", isNull(caseData.getFamilyManCaseNumber()) ? "" : caseData.getFamilyManCaseNumber() + ",")
```

Comment:
I'm tempted to maybe make a little private method that does a concat comma for strings when they are present

Reply:
Sam, do you think we could do this when we iterate in the area? I don't think we don't do it consistent way (how we send the data to the templates), would be happy to look at it in a holistic way.